runtime/lua/vim/bugreport/health.lua

```diff
@@ -0,0 +1,126 @@
+local M = {}
+
+local function detect_terminal()
+  if vim.env.TERM_PROGRAM then
+    local version = vim.env.TERM_PROGRAM_VERSION or ''
+    return vim.env.TERM_PROGRAM .. (version ~= '' and ' ' .. version or '')
+  end
+
+  if vim.env.KITTY_WINDOW_ID then
+    local result = vim.system({ 'kitty', '--version' }, { text = true }):wait()
+    if result.code == 0 then
+      local v = vim.version.parse(result.stdout, { strict = false })
+      return v and ('kitty ' .. tostring(v)) or 'kitty'
+    end
+    return 'kitty'
+  end
+
+  if vim.env.ALACRITTY_SOCKET or vim.env.ALACRITTY_LOG then
+    local result = vim.system({ 'alacritty', '--version' }, { text = true }):wait()
+    if result.code == 0 then
+      local v = vim.version.parse(result.stdout, { strict = false })
+      return v and ('alacritty ' .. tostring(v)) or 'alacritty'
+    end
+    return 'alacritty'
+  end
+
+  if vim.env.WEZTERM_EXECUTABLE then
+    local result = vim.system({ 'wezterm', '--version' }, { text = true }):wait()
+    if result.code == 0 then
+      local v = vim.version.parse(result.stdout, { strict = false })
+      return v and ('wezterm ' .. tostring(v)) or 'wezterm'
+    end
+    return 'wezterm'
+  end
+
+  if vim.env.KONSOLE_VERSION then
+    return 'konsole ' .. vim.env.KONSOLE_VERSION
+  end
+
+  if vim.env.VTE_VERSION then
+    return 'vte ' .. vim.env.VTE_VERSION
+  end
+
+  return 'unknown'
+end
+
+function M.check()
+  vim.health.start('Bug Report Information')
+
+  local version_output = vim.api.nvim_exec2('version', { output = true }).output
+  local nvim_version = version_output:match('NVIM v[^\n]+') or 'unknown'
```

Comment:
? `vim.version`

Reply:
because I can't get commit hash from vim.version 🤔

Reply:
Oh if build from package manager there just show name of it.. so can regex can as a fallback ?

```
{
  api_compatible = 0,
  api_level = 14,
  api_prerelease = true,
  build = "Homebrew",
  major = 0,
  minor = 12,
  patch = 0,
  prerelease = "dev",
  <metatable> = {
    __eq = <function 1>,
    __index = <function 2>,
    __le = <function 3>,
    __lt = <function 4>,
    __newindex = <function 5>,
    __tostring = <function 6>
  }
}
```

Reply:
oh. hmm... ok, let's use `:version` as you did then.

but add a 1-line comment explaining it:

    -- Use :version because vim.version().build returns "Homebrew" for brew installs.

Reply:
I’ll take a look at version.lua later and see if I can resolve it there ..